OracleAccessManagement/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,396 @@
+Building an OAM image with WebLogic Image Tool
```

Comment:
Please get your doc team to ensure this meets Oracle doc standards. You should also review your use of Markdown formatting to make sure things like environment variable names, filenames and paths use a monospace font.

Reply:
The documentation is still in process. Once done it should be with Oracle Standards.